nx-dev/ui-terms/src/lib/terms-and-definitions.tsx

```diff
@@ -0,0 +1,510 @@
+        will promptly reimburse Nx for any such amounts that Nx pays on your
+        behalf.
+      </p>
+
+      <h2>4. Confidential information</h2>
+      <p>
+        &quot;<strong>Confidential Information</strong>&quot; means any
+        information that one party (the &quot;
+        <strong>Disclosing Party</strong>&quot;) provides to the other party
+        (the &quot;<strong>Receiving Party</strong>&quot;) in connection with
+        this Agreement, whether orally or in writing, that is designated as
+        confidential or that reasonably should be considered to be confidential
+        given the nature of the information and/or the circumstances of
+        disclosure. Confidential Information will not include any information
+        that: (i) is or becomes generally known to the public through no fault
+        or breach of this Agreement by the Receiving Party; (ii) is rightfully
+        known by the Receiving Party at the time of disclosure without an
+        obligation of confidentiality; (iii) is independently developed by the
+        Receiving Party without access to or use of any Confidential Information
+        of the Disclosing Party that can be evidenced in writing; or (iv) is
+        rightfully obtained by the Receiving Party from a third-party without
+        restriction on use or disclosure. For clarity, the Software and the
+        Documentation will be deemed Confidential Information of Nx and Your
+        Data will be deemed your Confidential Information. The Receiving Party
+        will not use or disclose any Confidential Information of the Disclosing
+        Party except as necessary to perform its obligations or exercise its
+        rights under this Agreement; provided that Nx may use and modify your
+        Confidential Information in deidentified form for purposes of developing
+        and deriving Aggregated Data. The Receiving Party may disclose
+        Confidential Information of the Disclosing Party only: (A) to those of
+        its employees, contractors, agents and advisors who have a bona fide
+        need to know such Confidential Information to perform under this
+        Agreement and who are bound by written agreements with use and
+        nondisclosure restrictions at least as protective of the Confidential
+        Information as those set forth in this Agreement, or (B) as such
+        disclosure may be required by the order or requirement of a court,
+        administrative agency or other governmental body, subject to the
+        Receiving Party providing to the Disclosing Party reasonable written
+        notice to allow the Disclosing Party to seek a protective order or
+        otherwise contest the disclosure.
+      </p>
+      <h2>5. Policies</h2>
+      <p>
+        You hereby acknowledge that you have reviewed and agreed to the Nx
+        Privacy Policy at{' '}
+        <Link href="/privacy" prefetch={false} className="underline">
+          https://nx.app/privacy
```

Comment:
@juristr pending your confirmation
```suggestion
          https://nx.dev/privacy
```